include/deal.II/base/aligned_vector.h

```diff
@@ -292,16 +292,17 @@ namespace internal
      * Constructor. Issues a parallel call if there are sufficiently many
      * elements, otherwise work in serial. Copies the data from source to
      * destination and then calls destructor on the source. If the optional
-     * argument is set to true, the source is left untouched instead.
+     * argument copy_source is set to true, the source is left untouched
+     * instead.
```

Comment:
I think the term "untouched" is also misunderstandable -- you read the source, you just don't destroy it at the end. Maybe clarify this?

As a general remark about this function, when it talks about "source", I think what you might want to say is "every element in the range half open range from source_begin to source_end" or something similar.